src/components/EResourceSections/Agreements.js

```diff
@@ -32,16 +32,52 @@ export default class Agreements extends React.Component {
     visibleColumns: PropTypes.arrayOf(PropTypes.string),
   };
 
+  state = {
+    sortOrder: ['name', 'type'],
+    sortDirection: ['asc', 'desc'],
+  };
+
+  onSort = (e, meta) => {
+    if (!this.sortMap[meta.name]) return;
+
+    let {
+      sortOrder,
+      sortDirection,
+    } = this.state;
+
+    if (sortOrder[0] !== meta.name) {
+      sortOrder = [meta.name, sortOrder[0]];
+      sortDirection = ['asc', sortDirection[0]];
+    } else {
+      const direction = (sortDirection[0] === 'desc') ? 'asc' : 'desc';
+      sortDirection = [direction, sortDirection[1]];
+    }
+
+    this.setState({ sortOrder, sortDirection });
+  }
+
   renderEntitlementAgreements = () => {
     const { entitlements = [] } = this.props.data;
     const { headline, isEmptyMessage, visibleColumns } = this.props;
 
+    const {
+      sortOrder,
+      sortDirection,
+    } = this.state;
+
+    // eslint-disable-next-line no-undef
+    const contentData = _.orderBy(entitlements,
+      [this.sortMap[sortOrder[0]], this.sortMap[sortOrder[1]]], sortDirection);
+
     return (
       <EntitlementAgreementsList
-        entitlements={entitlements}
+        contentData={contentData}
```

Comment:
Any specific reason you changed the prop here Claudia?

Reply:
No, that (and the below) probably "happened" because I'm passing the `contentData`  from `Agreements` to `EntitlementAgreementsList`. I reverted the change.